zanata-war/src/main/java/org/zanata/events/ProjectUpdate.java

```diff
@@ -35,4 +35,6 @@
     public static final String EVENT_NAME = "org.zanata.events.ProjectUpdate";
 
     HProject project;
+
+    String oldSlug;
```

Comment:
Mmmm.... the events were removed, but the event classes are being modified here. I might be missing some new places where they are being used, but just in case adding this comment.